src/pydvl/value/semivalues.py

```diff
@@ -272,7 +272,7 @@ def compute_generic_semivalues(
                     # Filter out samples for indices that have already converged
                     filtered_samples = samples
                     if skip_converged and len(done.converged) > 0:
-                        # cloudpickle can't pickle this on python 3.8:
+                        # TODO: cloudpickle can't pickle this on python 3.8:
```

Comment:
I wouldn't say this is a to-do. It was just a comment explaining why I removed the nicer filter with a lambda

Reply:
I just added the TODO to make it easier to find once we stop supporting 3.8 which should happen next year.